utils/cligen/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (C) 2020-2023 Olof Hagsand and Rubicon Communications, LLC(Netgate)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cligen
+PKG_VERSION:=6.5.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Olof Hagsand <olof@hagsand.se>, Philip Prindeville <philipp@redfish-solutions.com>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/clicon/$(PKG_NAME)/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=8b3943430f7aa9eea6a5f7cf1ace5b68eb382380cf68f41ae3ef5e032e08816f
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE.md
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/cligen
+  SECTION:=utils
+  CATEGORY:=Utilities
+  URL:=https://www.cligen.se
+  TITLE:=CLIgen is a Command-Line Interface generator
+  DEPENDS:=libxml2
+endef
+
+define Package/cligen/description
+  CLIgen is a Command-Line Interface generator
```

Comment:
This is same as TITLE, imho, not good.

Reply:
Okay, will push a separate PR with updates shortly.

Reply:
Here: https://github.com/openwrt/packages/pull/22823